pkg/blackduck/blackduck.go

```diff
@@ -404,6 +410,26 @@ func (b *Client) GetPolicyStatus(projectName, versionName string) (*PolicyStatus
 	return &policyStatus, nil
 }
 
+func (b *Client) AddDefaultProjectTag(projectId string) error {
```

Comment:
Could you please add some unit tests for this function?

Reply:
Sure, I added them